program/tests/helpers/mint_builder.rs

```diff
@@ -0,0 +1,226 @@
+use {
+    crate::helpers::common::{init_mollusk, setup_mint},
+    mollusk_svm::{result::Check, Mollusk},
+    solana_account::Account,
+    solana_program::system_program,
+    solana_pubkey::Pubkey,
+    spl_token_wrap::{
+        get_wrapped_mint_address, get_wrapped_mint_backpointer_address, instruction::create_mint,
+    },
+};
+
+pub struct CreateMintResult {
+    pub unwrapped_mint: KeyedAccount,
+    pub wrapped_mint: KeyedAccount,
+    pub wrapped_backpointer: KeyedAccount,
+}
+
+#[derive(Debug, Clone)]
+#[cfg(test)]
+pub struct KeyedAccount {
+    pub key: Pubkey,
+    pub account: Account,
+}
+
+#[cfg(test)]
+impl KeyedAccount {
+    #[cfg(test)]
+    pub fn pair(&self) -> (Pubkey, Account) {
+        (self.key, self.account.clone())
+    }
+}
+
+#[derive(Debug, Clone, Copy)]
+pub enum TokenProgram {
+    SplToken,
+    Token2022,
```

Comment:
nit: token-2022 is also an SPL program, so let's be consistent and add the `Spl*` prefix to both, or neither

Reply:
Ah, I was wondering the right syntax for this. Will fix.